src/app/donation-start/donation-start.component.ts

```diff
@@ -373,6 +373,11 @@ export class DonationStartComponent implements AfterContentChecked, OnDestroy, O
       )
       .subscribe(async (donation: Donation) => {
         if (donation.psp === 'enthuse') {
+
+          // Because we're not storing the charity logo in the DB
```

Comment:
Given this is the case and it's not needed for the other mode, I think it would be better to add it as a second param for the Enthuse-specific method calls rather than adding it to the main `Donation` model.

Reply:
@NoelLH Okay, makes sense, i've passed in the uri as a 2nd param now and updated tests